homeassistant/components/abode/sensor.py

```diff
@@ -62,6 +66,11 @@ def device_class(self):
         """Return the device class."""
         return self._device_class
 
+    @property
+    def unique_id(self):
+        """Return a unique ID to use for this device."""
+        return self._device.device_uuid + self._sensor_type
```

Comment:
Please use f-strings instead of string concatenation.

Reply:
Done. Added a dash as well just so it's "uuid-sensortype", not that anyone will probably be looking at the uuid but just to make it a bit cleaner.